conf/messages.cy

```diff
@@ -450,4 +450,45 @@ ukInsuranceGains.none = No, I did not make any UK insurance gains
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingCloseCompanyLoans.p1 = A close company is a limited company with 5 or fewer ''participators'', or in which all the participators are also directors. If you get a loan or advance from a close company as a participator and the company releases or writes it off, it counts as your income.
+ukDividendsSharesAndLoansFromLimitedCompanies.legend = Did you get any UK dividends, shares or loans?
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.legend = Did you get any UK dividends, shares or loans?
+ukDividendsSharesAndLoansFromLimitedCompanies.cashDividendsUkStocksAndShares = Cash dividends from UK stocks and shares
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.cashDividendsUkStocksAndShares = Cash dividends from UK stocks and shares
+ukDividendsSharesAndLoansFromLimitedCompanies.stockDividendsUkCompanies = Stock dividends from UK companies
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.stockDividendsUkCompanies = Stock dividends from UK companies
+ukDividendsSharesAndLoansFromLimitedCompanies.dividends.hint = you must include all dividends except for dividends from ISAs
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.dividends.hint = you must include all dividends except for dividends from ISAs
+ukDividendsSharesAndLoansFromLimitedCompanies.dividendsUnitTrustsInvestmentCompanies = Dividends from unit trusts or investment companies
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.dividendsUnitTrustsInvestmentCompanies = Dividends from unit trusts or investment companies
+ukDividendsSharesAndLoansFromLimitedCompanies.freeOrRedeemableShares = Free or redeemable shares
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.freeOrRedeemableShares = Free or redeemable shares
+ukDividendsSharesAndLoansFromLimitedCompanies.closeCompanyLoansWrittenOffReleased = Close company loans written off or released
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.closeCompanyLoansWrittenOffReleased = Close company loans written off or released
+ukDividendsSharesAndLoansFromLimitedCompanies.noUkDividendsSharesOrLoans = No, I did not get any UK dividends, shares or loans
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.noUkDividendsSharesOrLoans = No, I did not get any UK dividends, shares or loans
+ukDividendsSharesAndLoansFromLimitedCompanies.exclusive = Exclusive
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.exclusive = Exclusive
+ukDividendsSharesAndLoansFromLimitedCompanies.checkYourAnswersLabel = Did you get any UK dividends, shares or loans?
+ukDividendsSharesAndLoansFromLimitedCompanies.error.required = Select if you had any UK dividends, shares or loans, or ''No, I did not get any UK dividends, shares, or loans''
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.error.required = Select if your client had any UK dividends, shares or loans, or ''No, my client did not have any UK dividends, shares or loans''
```

Comment:
For both of these errors we should use a different type of quotes like the other errors: or ‘No, I did not get any income from UK interest’.

Reply:
Yep thanks for pointing this out. I'll remove this in my next commit.